taskcluster/kinds/run-smoke-tests/kind.yml

```diff
@@ -25,3 +32,63 @@ tasks:
         mv ./ci_pyproject.toml ./pyproject.toml;
         pipenv install;
         pipenv run pytest --fx-executable ./firefox/firefox -n 4 .
+    notify:
+      recipients:
+        - type: slack-channel
+          channel-id: C07AHPJ525V
+          status-type: on-defined
+      content:
+        slack:
+          blocks: [
+                    {
+                      "type": "header",
+                      "text": {
+                        "type": "plain_text",
+                        "text": "firefox-desktop :firefox: ${task.metadata.name}\n "
+                      }
+                    },
+                    {
+                      "type": "divider"
+                    },
+                    {
+                        "type": "section",
+                        "text": {
+                            "type": "mrkdwn",
+                            "text": "*Task*: <https://firefox-ci-tc.services.mozilla.com/tasks/${status.taskId}|Taskcluster>"
+                      }
+                    },
+                    {
+                      "type": "section",
+                      "text": {
+                        "type": "mrkdwn",
+                        "text": "*Owner*: ${task.metadata.owner}"
+                      }
+                    },
+                    {
+                      "type": "section",
+                      "text": {
+                        "type": "mrkdwn",
+                        "text": "<https://github.com/mozilla/fx-desktop-qa-automation/commit/${task.metadata.source}|Commit>"
```

Comment:
looks like `task.metadata.source` is the full URL, so you don't need to add the base url to this link.

Reply:
Ok, let me look into it